### PR TITLE
Mc response to activity

### PR DESCRIPTION
**Pull request by mcharawi**
remove responseTo in favor of activityId

### CODE REVIEW

File: go/exocom/src/exocom/exocom.go [hunk 1]
```diff
@@ -140,14 +140,12 @@ func (e *ExoCom) send(message structs.Message) error {
...
 	}
-	if message.ResponseTo != "" {
-		originalTimestamp, ok := e.messageCache.Get(message.ID)
-		if ok {
-			message.ResponseTime = message.Timestamp.Sub(originalTimestamp)
-		}
-	} else {
-		e.messageCache.Set(message.ID, message.Timestamp)
+
+	originalTimestamp, ok := e.messageCache.Get(message.ID)
```
Comment (charlierudolph) [inline go/exocom/src/exocom/exocom.go]: We need to update the message cache to be a mapping from message.ActivityID to timestamp. The response time will be based on the last message with the same activityID.

**Comment by mcharawi** [comment]
All comments addressed @charlierudolph

File: go/exocom/src/logger/logger.go [hunk 2]
```diff
@@ -77,7 +78,7 @@ func (l *Logger) Messages(message structs.Message, originalName string, internal
 			text += fmt.Sprintf("--[ %s ]-[ %s ]->", originalName, internalMessageName)
 		}
 		text += "  " + receiver
-		if message.ResponseTo != "" {
+		if message.ResponseTime > time.Nanosecond {
```
Comment (charlierudolph) [inline go/exocom/src/logger/logger.go]: Is the default value 0? If so thoughts on comparing  `!= 0`? I think thats a bit more clear

Comment (mcharawi) [inline go/exocom/src/logger/logger.go]: The Duration type is technically just a wrapper for the int64 type, so the default is zero. Changed.

**Pull request merged**